README.md

```diff
@@ -231,71 +231,68 @@ Command above will build a vagrant box based on [fredrikhgrelland/bionic64-ansib
 
 **Consul**:  
-- Open source version
+- Using open source version
 - ACL [enabled=true](https://www.consul.io/docs/agent/options#acl_enabled)
 - [default_policy=allow](https://www.consul.io/docs/agent/options#acl_default_policy)
 
 **Nomad**:
-- Open source version
+- Using open source version
 - ACL [enabled=false](https://www.nomadproject.io/docs/configuration/acl#enabled)
-- [Integrated with Consul, using token](ansible/templates/nomad.hcl.j2)
-- [Integrated with Vault, using token](ansible/templates/nomad.hcl.j2)
+- [Integrated with Consul, using token](ansible/templates/conf/nomad/020-acl.hcl.j2)
+- [Integrated with Vault, using token](ansible/templates/conf/nomad/020-acl.hcl.j2)
 
 **Vault**
-- Open source version
-- Unsealed
-- PKI enabled at `/pki`
+- Using open source version
+- [Unsealed](https://www.vaultproject.io/docs/concepts/seal)
+- [PKI](https://www.hashicorp.com/products/vault/pki-with-vault) enabled at `/pki`
 
 ### Override default configuration
+To override the default configuration you have two options:
+- [Change the environment variables](#option-1---change-environment-variables)
+- [Override the configuration files](#option-2---override-configuration-files)
 
-#### Option 1 - env variables
-
-**Use env to switch prebuild configuration on/off**
+> :warning: Overriding config files will take effect at last. In other words, config files(Option 2) will override any configuration which were setup by the env variables(Option 1)
 
+#### Option 1 - Change environment variables
 When the vagrant box is provisioned, it reads the data from the following environment file `/home/vagrant/.env_default` in order to set up the system.
 If you wish to override any of the default values then you can do so by adding that variable name and value in [.env](https://github.com/fredrikhgrelland/vagrant-hashistack-template/blob/master/dev/.env) file.
 The property values in the `.env` file override the property values present in the `.env_default` file and thus makes it simple to provision systems that suffice the relevant development needs.
 
-
 For example, in order to override the **consul acl default policy** from **allow** to **deny**, the following needs to be added to the `.env` file:
 
 ```text
 consul_acl_default_policy=deny
 ```
 
-
-#### Option 2 - config files
-
-**Overriding config files***
-
-It is possible to add and/or override the hashistack components' configuration files. See documentation [here](https://github.com/fredrikhgrelland/vagrant-hashistack-template/tree/master/dev/vagrant/conf).
-
-`NB!` *Overriding config files will take effect at last. In other words, config files(Option 2) will override any configuration which were setup by the env variables(Option 1)*
+#### Option 2 - Override configuration files
+It is possible to add and/or override the hashistack components configuration files. See documentation [here](https://github.com/fredrikhgrelland/vagrant-hashistack-template/tree/master/dev/vagrant/conf).
```

Comment:
I think we need to override this section with -pre and -post procedures. 
It does not seem that we have direct access to the conf directories of services inside the box. [See here](https://github.com/fredrikhgrelland/vagrant-hashistack-template/pull/44/files)

Reply:
I'm not sure that I follow. Which file or files are you referring to in you link?

Reply:
Seems I misunderstood that PR. Nevermind